src/main/webapp/resources/js/pages/projects/redux/templatesSlice.js

```diff
@@ -66,23 +68,57 @@ export const createNewMetadataTemplate = createAsyncThunk(
   }
 );
 
+/**
+ * Redux Async Thunk for setting a default template for a specific project.
+ * @type {AsyncThunk<unknown, void, {}>}
+ */
+export const setDefaultTemplateForProject = createAsyncThunk(
```

Comment:
Why is this in here and not just in the projectSlice?

Reply:
Moved in 8e63a642f3f46bf750d3af026924c0835221c272